packages/agreement-process/src/services/agreementSubmissionProcessor.ts

```diff
@@ -65,17 +65,15 @@ export async function submitAgreementLogic(
   tenantQuery: TenantQuery
 ): Promise<Array<CreateEvent<AgreementEvent>>> {
   logger.info(`Submitting agreement ${agreementId}`);
-  const {
-    authData: { organizationId },
-  } = getContext();
+  const { authData } = getContext();
 
   const agreement = await agreementQuery.getAgreementById(agreementId);
 
   if (!agreement) {
     throw agreementNotFound(agreementId);
   }
 
-  assertRequesterIsConsumer(organizationId, agreement.data.consumerId);
```

Comment:
@Viktor-K another example of bug that could be prevented with branding.

This resulted in logging the wrong id when requester is not consumer and assertion fails: being the two params inverted, it was not logging the requesterId as expected, but the consumerId. Noticed it while testing this PR and fixed it here :)

Reply:
Good catch @ecamellini, branded string improvement was tracked with [IMN-166](https://pagopa.atlassian.net/browse/IMN-166) 👍🏻